src/article/shared/ManuscriptContentPackage.js

```diff
@@ -41,6 +41,7 @@ import {
   DefaultNodeComponent,
   ModelPreviewComponent,
 } from '../components';
+import { PersonComponent } from "../components/PersonComponent";
```

Comment:
If you export this in ../components/index.js, you could add this to the above import statement?